thangs_importer.py

```diff
@@ -250,8 +287,13 @@ def import_model(self):
                     self.file_path = os.path.join(self.temp_dir, self.model.partFileName)
                 else:
                     raise Exception("Unzipping didn't complete")
-        except:
+        except Exception as e:
             print('Unzip error')
+            self.amplitude.send_amplitude_event("Thangs Blender Addon - import model", event_properties={
```

Comment:
Do we want to include exception info in the amplitude event? I notice we're leaving it off here.

Reply:
Good catch.  I had tried that originally and was getting errors that the exception wouldn't serialize.  We had done that in other places in the code so I assumed it would work.  Figured out how to get it to work this morning.